README.md

```diff
@@ -58,6 +58,37 @@ sudo make install
 # install fzf: https://github.com/junegunn/fzf
 ```
 
+### Using oh-my-zsh
+
+Make sure that your oh-my-zsh `ZSH_CUSTOM` directory is configured,
+then clone navi into the plugins directory.
+
+```sh
+export ZSH_CUSTOM='/path/to/.zsh'
+
+DST="$ZSH_CUSTOM/plugins"
+mkdir -p "$DST" && cd "$DST"
```

Comment:
remove `&&` and add a new line (so that if the directory exists, it won't abort)

Reply:
`mkdir -p` succeeds even if the path already exists, so it won't abort. I can split it into two lines if you prefer, though.

Reply:
no strong opinions 👍